### ISSUE TITLE
Bug fixes

### ISSUE THREAD

**Issue by mt-** [breeko/Simple-Reinforcement-Learning-with-Tensorflow#1]
Thanks for doing this, there are a couple bugs to cleanup in Model-Based RL.ipynb (note the line numbers may be slightly off due to different spacing in my version):

Line #98: tf.concat( 1, [...] ) -> tf.concat( **[...], 1** )

Line #103: loss_done_m = -tf.log( actual_done_m * actual_done_m + ... ) -> loss_done_m = -tf.log( actual_done_m * **output_reward_m** + ... )

Line #194: next_states = states[1, :] -> next_states = states[1, **0:4**]

**Comment by breeko** [comment]
Thanks for checking out my repo!

Fixed the first one.

For the second one, I think you meant to write loss_done_m = -tf.log(actual_done_m * **output_done_m** + ...). I fixed that one as well.

For the third one, I want the action to be part of the state that I learn from. The reason is that the new state that the nn is learning from is partially determined by the action at the movement.

Also, I'm training the model nn and the policy nn separately. This doesn't really create a meaningful model as it doesn't experience longer games. I really need to toggle from training the policy and the model, which I have not done yet. It's doable, but just is a bit confusing to keep track of. I'll work on this in the next couple of days though.
